src/store/actions.ts

```diff
@@ -83,7 +83,10 @@ export default {
       const stateFilingUrl = rootGetters.getStateFilingUrl
 
       // if there is no state filing url, return null
-      if (!stateFilingUrl) resolve(null)
+      if (!stateFilingUrl) {
+        resolve(null)
+        return
```

Comment:
It appears that `resolve()` doesn't return. (I know if that's "sometimes" or "always", but the return statement is always safe.)

The bug was that line 91 ran even when no state filing was specified (empty), resulting in a console error. Also it's worth mentioned that this had no functional impact, since "no filing" was fetched when "no filing" was specified.

Reply:
Before:

![image](https://user-images.githubusercontent.com/10002889/235807082-becfc38d-8518-4b67-8db8-d7905dfa52b6.png)

After:

![image](https://user-images.githubusercontent.com/10002889/235807121-b88416fb-8a5a-4f84-b070-956f8e8d9f99.png)